SWLOR.Game.Server/Feature/SaveCharacters.cs

```diff
@@ -22,6 +23,13 @@ public static void HandleSaveCharacters()
                     if (GetLocalBool(player, "IS_BARTERING")) continue;
 
                     ExportSingleCharacter(player);
+
+                    // Clear combat state.
+                    if (!GetIsInCombat(player))
```

Comment:
Let's move this to the interval_pc_6s event since it's a completely separate thing from the saving process.

Reply:
Aha!  You already have an event for me to hook into, great.  Let me get that done.

Reply:
Done.